src/java.base/share/classes/java/lang/constant/ClassDesc.java

```diff
@@ -332,41 +271,13 @@ default ClassDesc componentType() {
      * default package, or this {@linkplain ClassDesc} does not describe a class or interface type
      */
     default String packageName() {
-        if (!isClassOrInterface())
-            return "";
-        String desc = descriptorString();
-        int index = desc.lastIndexOf('/');
-        return (index == -1) ? "" : internalToBinary(desc.substring(1, index));
+        return "";
     }
 
     /**
-     * Returns a human-readable name for the type described by this descriptor.
-     *
-     * @implSpec
-     * <p>The default implementation returns the simple name
-     * (e.g., {@code int}) for primitive types, the unqualified class name
-     * for class or interface types, or the display name of the component type
-     * suffixed with the appropriate number of {@code []} pairs for array types.
-     *
-     * @return the human-readable name
+     * {@return a human-readable name for this {@code ClassDesc}}
```

Comment:
```suggestion
     * {@return a human-readable name for this {@code ClassDesc}}
     * For primitive types, this method returns the simple name (for example {@code int}).
     * For class or interface types, this method returns the unqualified class name.
     * For array types, this method returns the human-readable name of the component 
     * type suffixed with the appropriate number of {@code []} pairs.
```

Reply:
Oops Mandy, just noticed that I forgot to push my latest commit. Pushed and I hope you can re-review.